presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkInjectorFactory.java

```diff
@@ -48,45 +48,27 @@
     private final Map<String, String> configProperties;
     private final Map<String, Map<String, String>> catalogProperties;
     private final Optional<Map<String, String>> eventListenerProperties;
+    private final Optional<Map<String, String>> accessControlProperties;
```

Comment:
Nice! We no longer need this ugly `accessControlModuleOverride`.

CC: @wenleix